ToDo.Entities/TodoTask.cs

```diff
@@ -5,6 +5,9 @@ namespace ToDo.Entities
     public class TodoTask : IUnique
     {
         public Guid Id { get; set; }
+        public bool IsCompleted { get; set; }
+        public string Name { get; set; }
         public string Description { get; set; }
+        public DateTime? CompletionDate { get; set; }
```

Comment:
I'm thinking this could be called `DueDate`, since `CompletionDate` evokes the actual time it was completed as opposed to the time this is supposed to be done by.